acquia_lift.admin.wizard.inc

```diff
@@ -582,21 +625,49 @@ function acquia_lift_personalize_campaign_wizard_review_alter(&$form, &$form_sta
 
     // Add audiences to the summary section.
   $agent_data = $form['#agent'];
+  if (isset($agent_data->data['variation_set_handling']) && $agent_data->data['variation_set_handling'] == ACQUIA_LIFT_DECISION_MULTIVARIATE) {
+    $form['review']['summary_column']['summary']['option_sets'] = array(
+      '#markup' => '<div class="personalize-wizard-summary-item clearfix"><span class="personalize-wizard-summary-count">1</span><div class="personalize-wizard-summary-details"><div class="item-list"><h3>Multivariate Test</h3></div></div></div>'
+    );
+  }
+  else {
+    // If there are multiple option sets, check that they all have the same number
+    // of options.
+    $option_sets = personalize_option_set_load_by_agent($agent_data->machine_name);
+    $equal_option_counts = TRUE;
+    if (count($option_sets) > 1) {
+      $first_os = array_shift($option_sets);
+      $num_options = count($first_os->options);
+      foreach ($option_sets as $os) {
+        if (count($os->options) != $num_options);
+        $equal_option_counts = FALSE;
+      }
+    }
+    // Prevent submission if the numbers of options aren't the same.
+    if (!$equal_option_counts) {
+      // @todo Get proper text for this.
+      drupal_set_message(t('You have multiple variation sets with different numbers of variations - the personalization cannot be started until all variation sets have the same number of variations.'), 'warning');
```

Comment:
We should show this message in the "Let's fix this" section - https://github.com/acquia/personalize_module/blob/workflow/personalize.admin.campaign.inc#L1158